### PR TITLE
8 implementation of users nearby ui

### PR DESCRIPTION
**Pull request by isym444**

### CODE REVIEW

File: dinnit/app/(tabs)/home.jsx [hunk 1]
```diff
@@ -1,12 +1,105 @@
...
+              <Image
+                source={user.avatar}
+                className="w-full h-full"
+              />
+            </View>
+          ))}
+          <View 
+            className="w-14 h-14 items-center justify-center"
+          >
+            <CirclePlus
```
Comment (Heroskin12) [inline dinnit/app/(tabs)/home.jsx:96]: All looks good but unsure if a plus circle is best choice here? Suggests adding more but the user can't add more to a 'nearby' section. Maybe a ... instead to suggest see more?

**Review by Heroskin12 — CHANGES_REQUESTED** [review]
left one comment in home.jsx

File: dinnit/app/(tabs)/home.jsx (hunk 1 above)
Comment (isym444) [reply dinnit/app/(tabs)/home.jsx:96]: ah cool, happy to add this instead, I was just following the wireframe 🤣. Will make this change to see how it looks and what we prefer :)

**Review by Heroskin12 — APPROVED** [review]
We can later export this to its own component maybe as well to keep the home file leaner. But otherwise all works and looks good.

**Pull request merged**